File: rb-appscript/tags/rb-appscript-0.2.0/sample/AB_list_people_with_emails.rb
#!/usr/bin/env ruby

# Lists the name and email(s) of every person in Address Book with one or more email addresses.

require "appscript"

people_ref = AS.app('Address Book').people[AS.its.emails.ne([])]
p people_ref.name.get.zip(people_ref.emails.value.get)

File: py-appscript/tags/py-appscript-0.17.3/Examples/appscript/AB_list_people_with_emails.py
#!/usr/bin/env python

# Lists the name and email(s) of every person in Address Book
# with one or more email addresses.

from appscript import *

peopleRef = app('Address Book').people[its.emails != []]
print zip(peopleRef.name.get(), peopleRef.emails.value.get())